### ISSUE TITLE
Playtime only updating ~every 30 minutes (while in game)

### ISSUE THREAD

**Issue by jayme-github** [jayme-github/steam_idle#5]
It looks as if the playtime is no longer update (at least for me) as long as I'm still "in game".
That concerns the profile page, the badges page as well as the gamecards page (like http://steamcommunity.com/my/gamecards/410210).

As soon as the game process terminates, the updated time is reflected on the pages.

**Comment by ghost** [comment]
I get random errors that "badge page is unavailable" or something like that, might be related. I'll post them once I see them again.

**Comment by jayme-github** [comment]
> I get random errors that "badge page is unavailable" or something like that, might be related. I'll post them once I see them again.

I don't think this is related (nevertheless you should open an issue for that) as I'm able to reproduce this with steam client itself (e.g. by actually launching a game at looking at the webpage for the playtime going up).

**Comment by ghost** [comment]
ah ok, sure I'll create an issue for it once I encounter the errors again.

**Comment by jayme-github** [comment]
Cool.
Would you be so kind and check back if the playtime is increased for you if you run a game? Further tests show that the playtime is updated in steam client (in library) but not on the pages and API. :disappointed:

**Comment by jayme-github** [comment]
Got confirmation of this from the SteamDB guys (thanks).
They said the playtime is now only updated every 30 minutes (I've not confirmed this yet) when a GamePlayed message is send.

**Comment by ghost** [comment]
Aha. Also I'm curious, is it supposed to update terminal view every second or so? Every second or so it writes a new line detailing the game being idled. Is that necessary or can I toggle that back?

**Comment by jayme-github** [comment]
> Aha. Also I'm curious, is it supposed to update terminal view every second or so? 

No it's not. I probably broke that some time ago. Created #6 for that.

**Comment by jayme-github** [comment]
Could probably work around the playtime update issue by sending GamePlayed messages on a regular basis via: https://bitbucket.org/AzuiSleet/pysteamkit or https://github.com/lunixbochs/vaporbat
